mkdocs.yml

```
@@ -3,7 +3,7 @@ site_author: crewAI, Inc
 site_description: Cutting-edge framework for orchestrating role-playing, autonomous AI agents. By fostering collaborative intelligence, CrewAI empowers agents to work together seamlessly, tackling complex tasks.
 repo_name: crewAI
 repo_url: https://github.com/joaomdmoura/crewai/
```

Can you also fix the repo url?
```suggestion
repo_url: https://github.com/crewAIInc/crewAI
```